docs/configuration/module-config/telemetry.mdx

```diff
@@ -103,11 +103,11 @@ All telemetry module config options are available in the python CLI. Example com
 
 |                         Setting                         |  Acceptable Values  | Default |
 | :-----------------------------------------------------: | :-----------------: | :-----: |
-|         telemetry.device_update_interval         | `integer` (seconds) |   `900`   |
+|         telemetry.device_update_interval         | `integer` (seconds) |   Default `0` is 15 minutes.   |
```

Comment:
Rather put 900 seconds here or at least add it, superficial readers may be tempted to set it to 20 (minutes) when they really wanna set 1200 (seconds).